### DISCUSSION TITLE
Sample usage explanation

### DISCUSSION THREAD

**Discussion by lawloretienne** [plhery/node-twitter-api-v2#306] [Q&A]
So based on my limited knowledge with full stack development, is this library intended to be used within the Node backend code within your project? What scope within a Javascript file should an import statement be declared like this : 

`import { TwitterApi } from 'twitter-api-v2';` 

If I declare that at the top level in a Javascript file in my Node backend code i get the following error
 
`SyntaxError: Cannot use import statement outside a module`.

 Is there any sample code within the project that shows where to position the code snippets into your code?

**Comment by SBTopZZZ-LG** [comment]
You can only use import statement inside modules. So in your case, instead of writing,
`import { TwitterApi } from 'twitter-api-v2';`
you can write,
`const { TwitterApi } = require('twitter-api-v2');`.

Alternatively, you can change the extension from ".js" to ".mjs" to mark the source code as an ES Module (which in this case you should be able to use the import statement without any issues, but you can no longer use the require as import already does it for you).
